pygsti/circuits/circuit.py

```diff
@@ -1789,7 +1857,7 @@ def delete_lines(self, lines, delete_straddlers=False):
                     raise ValueError(("Cannot remove a block that is straddled by "
                                       "%s when `delete_straddlers` == False!") % _Label(l))
             self._labels[i] = new_layer
-        self._line_labels = tuple([x for x in self.line_labels if x not in lines])
+        self.line_labels = tuple([x for x in self._line_labels if x not in lines])
```

Comment:
Another instance of LHS changing to property version. Keep as `_line_labels`?

Reply:
For reasons that I admittedly didn't fully understand at the time I had found that changing this and the similar one for tensoring broke some stuff in testing (i.e. in the unit tests). But, changing this again it looks like it isn't broken anymore. So I'm guessing I must have fixed whatever weird interaction was causing this at some point. 🤷‍♂️